test/fixtures/user/photo-verification.js

```diff
@@ -0,0 +1,30 @@
+const userPhotoVerificationData = {
+  discordId: "12345",
+  userId: "1234567abcd",
+  discord: {
+    url: "https://cdn.discordapp.com/avatars/abc/1234abcd.png",
+    approved: true,
+    date: "some-date",
```

Comment:
can we use the date format that will be actually used. instead of the random string.

Reply:
Sure